SourceX/controls/devices/joystick.cpp

```diff
@@ -6,26 +6,51 @@
 
 namespace dvl {
 
+namespace {
+
+bool IsModifierKey()
+{
+#ifdef JOY_MODIFIER_KEY
+	return SDLC_GetKeyState()[JOY_MODIFIER_KEY];

```

Comment:
Not sure this is right

Reply:
This should be `SDL_JoystickGetButton` https://www.libsdl.org/tmp/docs-1.2/_s_d_l__joystick_8h.html#af7789eb03bc42525691e075e2a36131